plugins/node/opentelemetry-koa-instrumentation/src/types.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Middleware, ParameterizedContext, DefaultState } from 'koa';
+import { RouterParamContext } from '@koa/router';
+import Router = require('@koa/router');
```

Comment:
Do you have to depend on `@koa/router`? Why require instead of regular import? Can this be 
```suggestion
import type * as Router from '@koa/router';
```